x-pack/plugins/spaces/server/routes/lib/parse_cloud_solution.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { SolutionView } from '../../../common';
+
+export const CLOUD_TO_KIBANA_SOLUTION_MAP = new Map<string, SolutionView>([
```

Comment:
__nit:__ looks like we don't use this outside of this file.
```suggestion
const CLOUD_TO_KIBANA_SOLUTION_MAP = new Map<string, SolutionView>([
```